### DISCUSSION TITLE
Welcome to dioxus Discussions!

### DISCUSSION THREAD

**Discussion by jkelleyrtp** [DioxusLabs/dioxus#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by c5soft** [comment]
Hi, Mr. Kelley, I am wondering what the meaning of the word Dioxus is. I known di means two and the Oxus is a river, today called Amu Darya. But what is di-oxus ? and the right pronunciation？ is [daɪˈɒksəs] ?

**Reply by jkelleyrtp** [reply]
Dioxus is pronounced like the two words you have. Di = Die, Oxus = aux us 

The Di roughly relates to the combination of frontend/backend with one project.
The Ox is related to Oxide (Rust)

The "us" doesn't really mean anything. I like the concept of "dual rivers" since that's roughly how things work.

**Reply by c5soft** [reply]
Interesting
